### DISCUSSION TITLE
usernames

### DISCUSSION THREAD

**Discussion by ShowtimeAnt** [sherlock-project/sherlock#957] [Q&A] [answered]
How come when I search somebody's instagram username, it doesn't say anything about instagram, it only says stuff about roblox and quiz something?

**Comment by DudeSavage** [comment]
I think that persons Instagram account doesn't exist anymore or it's
private account

**Comment by SethFalco** [comment, marked as answer]
At the time Instagram was disabled due to false-positives.

Instagram was brought back though on 25 Oct 2021 in https://github.com/sherlock-project/sherlock/commit/47c5ccd93e94f10aabe01e6c904a10af0657951f!
